samples/Bluetooth/BluetoothLESample1/Program.cs

```diff
@@ -0,0 +1,240 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+using nanoFramework.Device.Bluetooth;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+
+/// <summary>
+/// Bluetooth Sample 1 is a custom service which shows the use of:
+/// 
+/// - Read static value 
+/// - Read a dynamic value using an event 
+/// - Notifying clients of changed value
+/// 
+/// You will be able to connect to the service and read values or subscribe to Notified every 10 seconds.
+/// Suitable Phone apps: "LightBlue" or "nRF Connect"
+/// </summary>
+namespace BluetoothLESample1
+{
+    public class Program
+    {
+        static GattLocalCharacteristic _readCharacteristic;
+        static GattLocalCharacteristic _readWriteCharacteristic;
+
+        // Read/Write Characteristic value
+        static byte _redValue = 128;
+        static byte _greenValue = 128;
+        static byte _blueValue = 128;
+
+        public static void Main()
+        {
+            Debug.WriteLine("Hello from Bluetooth Sample 1");
+
+            // Define some custom Uuids
+            Guid serviceUuid = new Guid("A7EEDF2C-DA87-4CB5-A9C5-5151C78B0057");
+            Guid readCharUuid = new Guid("A7EEDF2C-DA88-4CB5-A9C5-5151C78B0057");
+            Guid readStaticCharUuid = new Guid("A7EEDF2C-DA89-4CB5-A9C5-5151C78B0057");
+            Guid readWriteCharUuid = new Guid("A7EEDF2C-DA8A-4CB5-A9C5-5151C78B0057");
+
+            //The GattServiceProvider is used to create and advertise the primary service definition.
+            //An extra device information service will be automatically created.
+            GattServiceProviderResult result = GattServiceProvider.Create(serviceUuid);
+            if (result.Error != BluetoothError.Success)
+            {
+                return;
+            }
+
+            GattServiceProvider serviceProvider = result.ServiceProvider;
+
+            // Get created Primary service from provider
+            GattLocalService service = serviceProvider.Service;
+
+            #region Static read characteristic
```

Comment:
very minor: you should have empty line before and after the region (applies in few other places), again, very minor